db/migrate/20170331194256_rename_users_recovery_code_to_personal_key.rb

```diff
@@ -0,0 +1,5 @@
+class RenameUsersRecoveryCodeToPersonalKey < ActiveRecord::Migration
+  def change
+    rename_column :users, :recovery_code, :personal_key
```

Comment:
If you're going to rename the db column, probably should also rename `recovery_cost` to `personal_key_cost` and `recovery_salt` to `personal_key_salt`.